_data/nav.yml

```diff
@@ -268,7 +268,6 @@
   - url: "comments"
```

Comment:
We've added new sections to all the plugin pages (`Menu Items` and `Toolbar Buttons`). Should they be included in the nav here?

We include `Commands` and `API` in most cases, so I think we should include these new sections.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1675/commits/9189463c6115d08e4d00e072ea557288f9416116

Leaving APIs, removed command headings from nav (for now).